common/netmap.go

```diff
@@ -0,0 +1,17 @@
+package common
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/interop/contract"
+	"github.com/nspcc-dev/neo-go/pkg/interop/runtime"
+)
+
+// SubscribeForNewEpoch registers calling contract as a NewEpoch
+// callback requester. Netmap contract's address is taken from the
+// NNS contract, therefore, it must be presented and filled with
+// netmap information for a correct SubscribeForNewEpoch call; otherwise
+// a successive call is not guaranteed.
```

Comment:
> what call is this line talking about?

method call

> not guaranteed seems like some kind of random

i meant it depends on an NNS implementation. it may and may not panic

Reply:
i've read this doc several times and not sure i completely understand it. We impose some preconditions (Netmap contract registered in NNS). If complied with - success, otherwise panic, right? Where is `not guaranteed` come from?

`filled with netmap information` - who must be filled and what information?

Reply:
> If complied with - success, otherwise panic, right?

well, to me that is not for sure. you need this to work it OK, but if it is not true, it depends on the NNS contract implementation. this call maybe a part of transaction, and panic (exception) may fail the whole TX

> Where is not guaranteed come from?

that is how i said "undefined behavior" but that time i wanted to say it this way

> filled with netmap information - who must be filled and what information?

NNS contract